xmppserver/src/main/java/org/jivesoftware/openfire/muc/spi/MUCPersistenceManager.java

```diff
@@ -76,6 +76,12 @@
         "canInvite, roomPassword, canDiscoverJID, logEnabled, subject, rolesToBroadcast, " +
         "useReservedNick, canChangeNick, canRegister, allowpm " +
         "FROM ofMucRoom WHERE serviceID=? AND (emptyDate IS NULL or emptyDate > ?)";
+    private static final String LOAD_REALLY_ALL_ROOMS =
```

Comment:
Can you think of a better name for this constant? Maybe rename to `LOAD_ALL_ROOMS` and rename the original query to `RELOAD_ALL_ROOMS_WITH_RECENT_ACTIVITY` or something like that.